server/protocols/webadmin/src/test/java/org/apache/james/webadmin/WebAdminConfigurationTest.java

```diff
@@ -46,7 +46,7 @@ public void buildShouldWorkWithoutPortWhenDisabled() {
         assertThat(WebAdminConfiguration.builder()
             .disabled()
             .build())
-            .isEqualTo(new WebAdminConfiguration(false, null));
+            .isEqualTo(new WebAdminConfiguration(false, null, HttpsConfiguration.builder().disabled().build()));
```

Comment:
that's the point where you need to use extract to check only flags you need to assert